FrankerFaceZ/FrankerFaceZ
# Add the workaround for combined emoji described in #1147.

This is a simple implementation of combined emoji. It does not account for skin colors when typing `:emoji_codes:` (👨🏼‍🌾 shows as `:man_farmer::tone_medium_light:` which isn't valid when sending). All it does is make sure that the emoji joiner gets replaced with the custom character and sent.

# Testing

- Open two windows (as to prevent client-side message rendering messing with you)
- Type a combined emoji like `👨🏼‍🌾`. It should appear as a single emoji on both windows. On vanilla Twilight it will appear as two on receiving side but as one on sending side.

Check list:
  - [x] Sending combined emoji
  - [x] Receiving combined emoji
  - [ ] Do I need to fill out some kind of changelog?

Closes #1147.
Side note: I intend to make PRs like this to other projects like Chatty and Chatterino so a lot of Twitch users can enjoy the wonders of Unicode.
